docs/apis-clients/tasklist-api/mutations/claim-task.mdx

```diff
@@ -3,15 +3,7 @@ id: claim-task
 title: claimTask
 ---
 
-Claim a task with taskId to currently logged in user. Returns the task.
```

Comment:
Removed change here as it needs to be done via a schema change as a separate PR for next versions.